### PR TITLE
Refinery Api Client

### PR DESCRIPTION
**Pull request by charmingduchess**
add refinery api client to the app
- caches data using python decorator @cache 
- invalidates cached data after 1 hr
- accepts fields `address` and `hours`

does not incorporate client into API.

### CODE REVIEW

File: lib/refinery_api.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        hours['today'] = True
+    if offset == 1:
+        hours['nextBusinessDay'] = True
+
+    return hours
+
+
+def build_hours_array(days_array, today):
+    # put sunday
+    refinery_days_sunday_last = days_array[1:] + days_array[0:]
```
Comment (nonword) [inline lib/refinery_api.py:122]: This assumes Sunday will always be first in Refinery data. Can we rely on that?

**Review by nonword — COMMENTED** [review]
Seems pretty good. What's missing?

File: lib/refinery_api.py (hunk 1 above)
Comment (charmingduchess) [reply lib/refinery_api.py:122]: every response i've seen has had the same order. not sure if there is any written documentation that describes that behavior. i can add something to ensure it's in the correct order if you'd prefer that.

**Comment by charmingduchess** [comment]
@nonword I must have left it as a draft request by accident. I think it's all there!

File: lib/refinery_api.py (hunk 1 above)
Comment (nonword) [reply lib/refinery_api.py:122]: Given that there is likely to be no further development on Refinery, let's assume it's a stable thing.

**Review by nonword — APPROVED** [review]

**Pull request merged**